app/screens/ChildSetup.tsx

```diff
@@ -313,26 +340,36 @@ const ChildSetup = ({ navigation }: Props): any => {
       setIsImportRunning(false);
     }
   }
-
-
-  const AddChild = async (): Promise<any> => {
+  const AddChild = async (isDefaultChild: boolean): Promise<any> => {
     await userRealmCommon.getData<ChildEntity>(ChildEntitySchema);
-    const defaultName = name;
+    let defaultName ;
+    if(isDefaultChild){
+       defaultName=' Baby';

```

Comment:
Did you leave space intentionally?

Reply:
No, actually here I will use string from string file.